tests/rust/re_integration_test/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+use std::process::{Child, Command, Stdio};
+
+pub struct TestServer {
+    server: Child,
+}
+
+impl TestServer {
+    pub fn spawn() -> Self {
+        let mut build = Command::new("pixi");
+        build.args(["run", "rerun", "--help"]);
+        build.stdout(Stdio::null());
+        build.spawn().unwrap().wait_for_success();
```

Comment:
Probably more clear:

```suggestion
        // First build the binary:
        let mut build = Command::new("pixi");
        build.args(["run", "rerun-build"]);
        build.stdout(Stdio::null());
        build.spawn().unwrap().wait_for_success();
```

Though we need to add the `oss_server` feature to https://github.com/rerun-io/rerun/blob/1055e0026c71014a5d28c8092a7b2d0c5329cdda/pixi.toml#L186